pkg/files/files.go

```diff
@@ -143,6 +143,7 @@ func ReadLines(content string) []string {
 
 // GetContentType returns the content type of a file
 func GetContentType(path string, config config.Config) (string, error) {
+
```

Comment:
Could you remove this empty added line?

Reply:
Yeah, sorry. That got added by accident.